src/web/mjs/connectors/PeanuToon.mjs

```diff
@@ -0,0 +1,190 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class PeanuToon extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'peanutoon';
+        super.label = 'Peanutoon (피너툰)';
+        this.tags = [ 'webtoon', 'korean' ];
+        this.url = 'https://www.peanutoon.com';
+        this.config = {

```

Comment:
While we are at it, since this website requires login, you could add the login link in the constructor.
Look how its done for other websites.

this.links = {
    login : 'http:/wqsdqsq'
};

Providing its another page and not a popup on the same page.

Reply:
Done!